ec_ms_quantification/EC-MS_ixdat_tutorial.ipynb

```diff
@@ -0,0 +1,1017 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "49dd450c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "axes_a = meas.plot_measurement(tspan=(0, 3200))\n",
+    "fig_a = axes_a[0].get_figure()  # name the figure by getting it via the first axis.\n",
+    "fig_a.set_figwidth(\n",
+    "    fig_a.get_figwidth() * 2.5\n",
+    ")  # make it 2.5 times as wide as it is tall"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5c1e4ba",
+   "metadata": {},
+   "source": [
+    "Great! Now we're ready to use this data to calibrate for H2 at m/z=2. For this we use the ixdat method ``ecms_calibration_curve``, which automatically selects and integrates M2 signal and electrochemical current in the given timespans. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b3cfe7e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# redefine this axis, so we can use it here (since Jupyter doesn't update already printed plots).\n",
+    "axes_a = meas.plot_measurement(tspan=(0, 3200)) \n",
+    "# calculate the calibration factor\n",
+    "cal_result_H2, ax_b = meas.ecms_calibration_curve(\n",
+    "    mol=\"H2\",\n",
+    "    mass=\"M2\",\n",
+    "    n_el=-2, # remember to use the correct sign: minus for reduction reactions, plus for oxidation reactions\n",
+    "    tspan_list=[(450, 500), (1000, 1050), (1550, 1600), (2200, 2250), (2750, 2800)],  # timespans of H2 evolution\n",
+    "    tspan_bg=(0, 100),\n",
+    "    ax=\"new\",\n",
+    "    axes_measurement=axes_a, # to highlight the integrated areas on the plot defined above\n",
+    "    return_ax = True # if True, returns the calibration curve axis as a second element\n",
+    ") \n",
+    "# NOTE, it uses and highlights the electrochemical current, not the calculated current density - FIXME!!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "53d6f687",
+   "metadata": {},
+   "source": [
+    "Because we asked for ``return_ax``, two arguments are returned:\n",
+    "The first, which we call ``cal_result_H2``, is a ``MSCalResult`` for H2.\n",
+    "The second, which we call ``ax_H2``, is the axis where it plots the calibration curve.\n",
+    "As we asked for ``axes_measurement``, to be plotted on ``axes_a``, the areas that were integrated are highlighted on that axes.\n",
+    "\n",
+    "The attribute ``cal_result_H2.F`` is the slope of the calibration curve, which is the sensitivity factor in C/mol.\n",
+    "(Note that this calibration_curve works with integrals rather than simply the rates. The latter is not yet implemented in ixdat.)"
```

Comment:
I see only one reason why this could be of interest would be if I wanted to plot calibration curves for ECMS calibration and gas flux calibrations in one plot. So I might be the only person who's interested in this. Added this as a new issue #123